### PR TITLE
Migrate no_dupe_args from Visitor to Handler

### PR DESCRIPTION
**Pull request by DmitryRomaniuk**
https://github.com/denoland/deno_lint/issues/818

### CODE REVIEW

File: src/rules/no_dupe_args.rs [hunk 1]
```diff
@@ -106,22 +97,20 @@ impl<'c, 'view> NoDupeArgsVisitor<'c, 'view> {
...
 }
 
-impl<'ctx, 'view> VisitAll for NoDupeArgsVisitor<'ctx, 'view> {
-  noop_visit_type!();
-
-  fn visit_function(&mut self, function: &Function, _parent: &dyn Node) {
-    self.check_params(function.span, function.params.iter());
+impl Handler for NoDupeArgsHandler {
+  fn function(&mut self, function: &Function, _ctx: &mut Context) {
+    self.check_params(function.span(), function.params.iter());
```
Comment (DmitryRomaniuk) [inline src/rules/no_dupe_args.rs:109]: alternative is move map from check_params and delete check_params
```suggestion
    self.check_pats(function.span(), function.params.iter().map(|param| &param.pat));
```

File: src/rules/no_dupe_args.rs [hunk 2]
```diff
@@ -106,22 +97,20 @@ impl<'c, 'view> NoDupeArgsVisitor<'c, 'view> {
 
   fn check_params<'a, 'b, I>(&'a mut self, span: Span, params: I)
   where
```
Comment (DmitryRomaniuk) [inline src/rules/no_dupe_args.rs:99]:
```suggestion
```

File: src/rules/no_dupe_args.rs (hunk 1 above)
Comment (magurotuna) [reply src/rules/no_dupe_args.rs:109]: Either looks fine to me

**Review by magurotuna — APPROVED** [review]
LGTM

**Review by bartlomieju — APPROVED** [review]

**Pull request merged**